content/en/methods/streaming.md

```diff
@@ -668,7 +675,7 @@ An example filter change by the user:
 ```
 
 {{< hint style="warning" >}}
-Note that the `payload` property is not present for `filters_changed` events.
+Note that the `payload` property is not present for `filters_changed` events. And for `delete` and `announcements.delete` the payload is a string, not an object.
```

Comment:
redundant addition here

Reply:
Was deliberate, since it's easily missed and one of the bugs to fix in v2 of streaming.